packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -190,6 +193,28 @@ export const getStandardFieldConfigs = () => {
     category,
   };
 
+  const nullValue: FieldConfigPropertyItem<any, NullValueMode, SelectFieldConfigSettings<NullValueMode>> = {
+    id: 'nullValueMode',
+    path: 'nullValueMode',
+    name: 'Null values',
+    description: 'Behavior when data includes null values',
+
+    editor: standardEditorsRegistry.get('radio').editor as any,
+    override: standardEditorsRegistry.get('radio').editor as any,
+    process: selectOverrideProcessor,
+
+    settings: {
+      options: [
+        { label: 'Gaps', value: NullValueMode.Null },
+        { label: 'Connected', value: NullValueMode.Ignore, description: 'Ignore the values from any calculation' },
```

Comment:
"Connected" always seemed odd to me, the nulls themselves are not connected, they're "Spanned" or Ignored

Reply:
Yea agree, ignored / skipped seems better